src/system/ui/pages/comment/actions.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import { getWpRest } from 'utils/wordpress'
+import { Icon } from 'ui'
+import { getSystemActions } from 'data'
+
+export const getCommentActions = ( {
+	set_approveStatus, set_responseMessage, set_spamStatus, set_trashStatus, set_commentStatus, item } ) => {
+
+	const {
+		id,
+		url,
+		editUrl,
+		approved,
+		spam,
+		trash,
+	} = item
+	const comments = getWpRest()
+	const { setCurrentHistoryState } = getSystemActions()
+	const [ actionApprove, set_actionApprove ] = React.useState( approved )
+	const [ actionSpam, set_actionSpam ] = React.useState( spam )
+	const [ actionTrash, set_actionTrash ] = React.useState( trash )
```

Comment:
Variables should be camel case, even state setters.

Reply:
Ok! Renamed variables to camel case.